parachains/runtimes/collectives/collectives-polkadot/src/fellowship/migration.rs

```diff
@@ -0,0 +1,260 @@
+		}
+
+		#[cfg(feature = "try-runtime")]
+		fn post_upgrade(state: Vec<u8>) -> Result<(), &'static str> {
+			assert_eq!(MemberCount::<T, I>::get(0), 45, "invalid members count at rank 0.");
+			assert_eq!(MemberCount::<T, I>::get(1), 45, "invalid members count at rank 1.");
+			assert_eq!(MemberCount::<T, I>::get(2), 24, "invalid members count at rank 2.");
+			assert_eq!(MemberCount::<T, I>::get(3), 17, "invalid members count at rank 3.");
+			assert_eq!(MemberCount::<T, I>::get(4), 10, "invalid members count at rank 4.");
+			assert_eq!(MemberCount::<T, I>::get(5), 7, "invalid members count at rank 5.");
+			assert_eq!(MemberCount::<T, I>::get(6), 3, "invalid members count at rank 6.");
+			assert_eq!(MemberCount::<T, I>::get(7), 0, "invalid members count at rank 7.");
+			Ok(())
+		}
+	}
+}
+
+#[cfg(test)]
+pub mod tests {
+	use super::import_kusama_fellowship::FellowshipAddresses;
+	use crate::{FellowshipCollectiveInstance as Fellowship, Runtime, System};
+	use frame_support::traits::OnRuntimeUpgrade;
+	use pallet_ranked_collective::Rank;
+	use parachains_common::AccountId;
+	use sp_core::crypto::Ss58Codec;
+	use sp_runtime::AccountId32;
+
+	#[test]
+	fn check_fellowship_addresses() {
+		let fellowship_addresses = FellowshipAddresses::get();
+		let kusama_fellowship_ss58: [(Rank, _); 45] = [
+			(6, "FcxNWVy5RESDsErjwyZmPCW6Z8Y3fbfLzmou34YZTrbcraL"),
+			(6, "J8ww78Qx3LVLW54bva3t4SzXcWKMdUWHEZR3V2VNKbmQgE8"),
+			(6, "FFFF3gBSSDFSvK2HBq4qgLH75DHqXWPHeCnR1BSksAMacBs"),
+			(5, "G7YVCdxZb8JLpAm9WMnJdNuojNT84AzU62zmvx5P1FMNtg2"),
+			(5, "FczL661VujsMFqZT32Y3iBmNpSxUw4gMnq2JtdoFT6rJzr4"),
+			(5, "Dcm1BqR4N7nHuV43TXdET7pNibt1Nzm42FggPHpxKRven53"),
+			(5, "J9nD3s7zssCX7bion1xctAF6xcVexcpy2uwy4jTm9JL8yuK"),
+			(4, "EGVQCe73TpFyAZx5uKfE1222XfkT3BSKozjgcqzLBnc5eYo"),
+			(4, "HL8bEp8YicBdrUmJocCAWVLKUaR2dd1y6jnD934pbre3un1"),
+			(4, "HyBryanRsB1GGKa9ZfqvRc3XpTDipYyRvxNNyZYfWFcenhd"),
+			(3, "GAWwkmjbLhM5pnAVbdZEkwd3QjHE7kaxJSCi3Ec91Q3QSDW"),
+			(3, "H25aCspunTUqAt4D1gC776vKZ8FX3MvQJ3Jde6qDXPQaFxk"),
+			(3, "GtLQoW4ZqcjExMPq6qB22bYc6NaX1yMzRuGWpSRiHqnzRb9"),
+			(3, "G5MVrgFmBaYei8N6t6DnDrb8JE53wKDkazLv5f46wVpi14y"),
+			(3, "HfFpz4QUxfbocHudf8UU7cMgHqkHpf855Me5X846PZAsAYE"),
+			(3, "DSbhnaGBytDGRfZTmdcArzCL6T3HQ8gcZxWpF5gLBP6y1Qe"),
+			(3, "CzEPpMr7XNS6dK7nQFnQbfnJQYLq7nvULK5kL9U8Zb6CTJm"),
+			(2, "Ddb9puChKMHq4gM6o47E551wAmaNeu6kHngX1jzNNqAw782"),
+			(2, "GcDZZCVPwkPqoWxx8vfLb4Yfpz9yQ1f4XEyqngSH8ygsL9p"),
+			(2, "GA3yPifemubFga7sTSFtLY2KFFiSRp6Bb8w31FS4xqgAvCz"),
+			(2, "HxhDbS3grLurk1dhDgPiuDaRowHY1xHCU8Vu8on3fdg85tx"),
+			(2, "HTk3eccL7WBkiyxz1gBcqQRghsJigoDMD7mnQaz1UAbMpQV"),
+			(2, "EcNWrSPSDcVBRymwr26kk4JVFg92PdoU5Xwp87W2FgFSt9c"),
+			(2, "D8sM6vKjWaeKy2zCPYWGkLLbWdUtWQrXBTQqr4dSYnVQo21"),
+			(1, "GfbnnEgRU94n9ed4RFZ6Z9dBAWs5obykigJSwXKU9hsT2uU"),
+			(1, "HA5NtttvyZsxo4wGxGoJJSMaWtdEFZAuGUMFHVWD7fgenPv"),
+			(1, "Hj44XnjZui7SQ3A5eBMoJFa4H4nVhiyWnL2i2xw5f1YqzRX"),
+			(1, "Ea6jhP5gF4r7NqhkEoAXJDgSgYpNQNaTYU6gPsrEGfctaKR"),
```

Comment:
```suggestion
			(1, "16a357f5Sxab3V2ne4emGQvqJaCLeYpTMx3TCjnQhmJQ71DX"),
```
https://kusama.subscan.io/extrinsic/16836396-5  
![image](https://user-images.githubusercontent.com/10380170/221939706-88d6d60a-f7d0-42b7-aa27-944fda077fb1.png)